AnomalyDetection.py

```diff
@@ -1,18 +1,21 @@
-import math, matplotlib, bisect
+import math, bisect
```

Comment:
nit: imports should go on separate lines
https://www.python.org/dev/peps/pep-0008/#imports

Reply:
I know that but I didn't particularly want that requirement. So I left it. If you or someone really wants to conform to PEP8 100%, I will be OK with that.